### PR TITLE
Fixes #29398 - Correct errata redirection from repo

### PR DESCRIPTION
**Pull request by sjha4**

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js [hunk 1]
```diff
@@ -18,16 +18,21 @@
...
+    function ($scope, $state, $stateParams, $location, translate, Nutupane, Erratum, IncrementalUpdate, Repository, CurrentOrganization) {
         var nutupane, params = {
             'organization_id': CurrentOrganization,
             'search': $location.search().search || "",
             'sort_by': 'updated',
             'sort_order': 'DESC',
             'paged': true,
-            'errata_restrict_applicable': true
+            'errata_restrict_applicable': false,
+            'disableAutoLoad': true
```
Comment (ianballou) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js]: Why did you need to set `disableAutoLoad`?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js [hunk 2]
```diff
@@ -37,21 +42,24 @@ angular.module('Bastion.errata').controller('ErrataController',
         Erratum.queryPaged({'organization_id': CurrentOrganization}, function (result) {
             $scope.errataCount = result.total;
         });
-
```
Comment (ianballou) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js]: Removed a white space line?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js (hunk 1 above)
Comment (sjha4) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/errata.controller.js]: I was seeing 2 API calls being made to errata controller on opening the page. This flag tells nutupane not to load the records when new Nutupane is called. Instead wait till all params are filled in the controller and load when refresh is called towards the end.
